util/hypercall_external_signal/orchestrator-request.py

```diff
@@ -171,7 +171,9 @@ def receive_full_message(conn: socket.socket, timeout: float = 30.0) -> str:
     return b"".join(chunks).decode()
 
 
-def send_and_receive_hypercall(pid: int, function: str) -> str:
+def send_and_receive_hypercall(
+    pid: int, function: str, debug_flags_to_add: str = None
```

Comment:
I don't think it makes sense to have an argument specific to one function in the generic function. This can be solved by either (1) making it a generic "arguments" or (2) making a new function to send/recv "add_debug_flags".

Also, why is it called "add_debug_flags" if you can both enable and disable flags? Maybe it should be "update_debug_flags" or something like that.